compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/NativeDependencyResolver.java

```diff
@@ -0,0 +1,21 @@
+package org.wso2.ballerinalang.compiler;
+
+import org.ballerinalang.model.elements.PackageID;
+
+import java.nio.file.Path;
+import java.util.List;
+
+/**
+ * Contains methods to resolve the module jars and native libraries.
+ *
+ * @since 1.3.0
```

Comment:
Since 1.3.0 is an old release I was wondering if this is correct. Same for the `NativeDependencyResolverImpl` class.

Reply:
Yeah true. As per the current release road map there will be no 1.3.0 release. The next major release will be 2.0.0.